libraries/botbuilder/src/teamsActivityHandler.ts

```diff
@@ -133,9 +133,6 @@ export class TeamsActivityHandler extends ActivityHandler {
                             await this.handleTeamsTabSubmit(context, context.activity.value)
                         );
 
-                    case 'adaptiveCard/action':
```

Comment:
Note: Everything removed in this file and related tests is already covered on new line 138, when the `default` of the switch/case calls `super.onInvokeActivity()`